py4cast/datasets/poesy.py

```diff
@@ -309,6 +384,7 @@ def __init__(
         self.shuffle = self.split == "train"
         self.cache_dir.mkdir(parents=True, exist_ok=True)
         self.step_duration = self.settings.term["timestep"]
```

Comment:
I think we should decouple `step_duration` from `term["timestep"]`.
The former is a parameter we might want to vary in the experiments, the latter simply describes how poesy samples are arranged. Later PR, probably,  but to keep in mind.

Reply:
--> done in later PR